contracts/interfaces/ILimitOrder.sol

```diff
@@ -19,6 +39,17 @@ interface ILimitOrder is IStrategyBase {
         address recipient,
         FillReceipt fillReceipt
     );
+
+    /// @notice Emitted when an order is filled by interacting with an external protocol
+    /// @param orderHash The EIP-712 hash of the target order
+    /// @param maker The address of the maker
+    /// @param taker The address of the taker (trader)
+    /// @param allowFillHash The EIP-712 hash of the fill permit granted by coordinator
+    /// @param relayer The address of the relayer
+    /// @param profitRecipient The address of the recipient which receive profit for relayer
```

Comment:
nit:
```suggestion
    /// @param profitRecipient The address of the recipient which receives the relayer profit
```

Reply:
Adjusted in 7a91187197c37a8285b47ee4f3227bad434ebccb.